client/actions/user-action-creators.js

```diff
@@ -2,7 +2,7 @@ import * as userApi from 'api/user-api';
 import { push } from 'react-router-redux';
 import userActionTypes from './user-action-types';
 
-export function facebookLoginSubmitted(user) {
+export default function facebookLoginSubmitted(user) {
```

Comment:
This is breaking the facebook login. When you exporting `default` only exports that  one thing from a file. I'm guessing your linter was yelling at you to change this? All action-creator files will have multiple exports, so this warning you can always ignore when you're in an action creator file or a file that you know will have many exported functions.